url_encoder/plugin.spec.yaml

```
@@ -0,0 +1,70 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: url_encoder
+title: URL Encoder
+description: Encode and decode URLs
+version: 1.0.0
+vendor: rapid7
+support: community
+status: []
+cloud_ready: true
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/url_utils
```

This needs to be updated to match the new plugin name
```suggestion
  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/url_encoder
```